install_config/configuring_kuryrsdn.adoc

```diff
@@ -69,10 +69,9 @@ Kuryr-Kubernetes as the network plug-in:
  # Set userspace so that there are no iptables remains.
  openshift_node_proxy_mode='userspace'
 
- # Disable management of the OpenShift Registry and OpenShift Router. If you
- # need those services, install them manually after OpenShift deployment using
- # `oadm`. This way services will get correct IPs provided by Kuryr.
- openshift_hosted_manage_registry=false
+ # Disable management of the OpenShift Router. If you need this service, install
```

Comment:
why you need to disable router? I believe there is no need to disable registry, nor router right now

Reply:
Ah, okay! I wasn't sure. I'll update.